### PR TITLE
status and logging

### PR DESCRIPTION
**Pull request by mdip226**
added logfile parameter and changed prints to logging.info()

### CODE REVIEW

**Review by BryanRumsey — CHANGES_REQUESTED** [review]
- Need to setup a stochss compute logger.
- Need additional status updates.

**Comment by mdip226** [comment]
rename EC2Cluster

**Review by BryanRumsey — APPROVED** [review]

File: stochss_compute/cloud/ec2.py [hunk 1]
```diff
@@ -44,19 +59,33 @@ class Cluster(Server):
     _server = None
     _ami = None
 
-    def __init__(self) -> None:
+    def __init__(self, statusFile=None) -> None:
```
Comment (briandrawert) [inline stochss_compute/cloud/ec2.py:62]: To conform with Python standards, we should use 'snake case' not 'camel case' where we can.  Please change this to "status_file"

File: stochss_compute/cloud/ec2.py [hunk 2]
```diff
@@ -44,19 +59,33 @@ class Cluster(Server):
...
         """ 
         Attempts to load a StochSS-Compute cluster. Otherwise just initializes a new cluster.
+
+        :param statusFile: Optional. If provided, status updates will be written to a file with this path and name.
```
Comment (briandrawert) [inline stochss_compute/cloud/ec2.py:66]: and here

File: stochss_compute/cloud/ec2.py [hunk 3]
```diff
@@ -44,19 +59,33 @@ class Cluster(Server):
...
+        :type str:
+
         """
+        self.statusFile = statusFile
```
Comment (briandrawert) [inline stochss_compute/cloud/ec2.py:70]: and here

File: stochss_compute/cloud/ec2.py [hunk 4]
```diff
@@ -71,23 +100,46 @@ def address(self):
...
+    @property
+    def status(self):
+        if self._server is None:
+            return self._status
+        else:
+            return self._server.state['Name']
+
+    def _set_status(self, status):
+        self._status = status
+        if self.statusFile is not None:
```
Comment (briandrawert) [inline stochss_compute/cloud/ec2.py:112]: and here

**Comment by mdip226** [comment]
> This might not be the right place to discuss, but a "sssc-root.pem" file is being created. This should be configured to be in a different location. Ask @BryanRumsey for more info

I'm gonna work on that in #164 that way I can organize the changes into one chunk. Basically I'm gonna pull out the constants from the ec2 file and make them configurable.

**Review by BryanRumsey — APPROVED** [review]

**Review by briandrawert — APPROVED** [review]

**Pull request merged**